### ISSUE TITLE
FootprintService Error 

### ISSUE THREAD

**Issue by jaumard** [trailsjs/trailpack-waterline#4]
I'm trying to use this : 

```
this.api.services.FootprintService.update('User',
      {id: 1},
      {username: 'updated'}).then(function (elements) {
      res.status(201).json(elements)
    }).catch(function (error) {
      res.status(500).json(error)
    })
```

But it's not working... I have : 

```
{"error":"E_UNKNOWN","status":500,"summary":"Encountered an unexpected error","raw":{"errno":1,"code":"SQLITE_ERROR"}}
```

This problem occur also with `findAssociation` and `updateAssociation`, other methods seems to work. 
I run test unit of trailpack-waterline and there no problem for the update method... 

How can fix this ? Or have a better error message.

**Comment by jaumard** [comment]
Fix on version 0.0.7 (master) but not yet on npm @tjwebb can you publish it ?

**Comment by jaumard** [comment]
Fixed
